book/content/open-source-comms/comms-channels.md

```diff
@@ -0,0 +1,34 @@
+# Communication Channels
+
+The channel(s) through which you choose to communicate with your community will have a big impact on the experience and sense of belonging for your community members.
+The choice of channels will depend on the specific requirements of your project, such as keeping the community informed of any known bugs or upcoming features, assessing the current needs from the community members, inviting comments on the management plans, facilitating the overall progress of the project, as well as having somewhere to field questions and feedback.
+
+When setting up your communication channels, there are some important things to consider such as the following:
+
+* How easy is it for the community to find these channels and subscribe/unsubscribe to them?
```

Comment:
Moderation - Under-represented groups may not participate if a channel is a free for all.